udata/harvest/backends/dcat.py

```diff
@@ -1,19 +1,17 @@
 import logging
```

Comment:
I've just tested it on a csw-iso-19139 harvester on https://data.naturefrance.fr/geonetwork/srv/eng/csw with an added
```
                                    <ogc:PropertyIsEqualTo>
                                        <ogc:PropertyName>dc:type</ogc:PropertyName>
                                        <ogc:Literal>service</ogc:Literal>
                                    </ogc:PropertyIsEqualTo>
```
:arrow_up: this filter should be added if we can make it work properly.

The two dataservices get harvested, but other erroneous items get created on the way.

Reply:
It's 7 unique datasets (but 28 items during harvesting, why x4? I don't know) referenced inside `servesDataset`. The catalog is just listing the dataset without any information inside it.